src/config.ts

```diff
@@ -3,7 +3,7 @@ import { promises as fs } from 'fs';
 import * as os from 'os';
 import * as path from 'path';
 
-export type ToolType = 'cargo' | 'go' | 'benchmarkjs' | 'pytest' | 'googlecpp' | 'catch2';
+export type ToolType = 'cargo' | 'go' | 'benchmarkjs' | 'pytest' | 'googlecpp' | 'catch2' | 'roblox';
```

Comment:
can we call this `lua`? then we can upstream the change more readily

Reply:
well, this applied to all of our benchmarks. Both the cachegrind and regular FPS and Δt benchmarks so I don't think cachegrind is appropriate. I don't think that `lua` is a great choice as well as we combined both runtime and cachegrind benchmarks together so in order to make it work we would have to split the benchmark results which would cause readability issues IMO.
This applies a lot of changes to the structure of the code so I don't think (judging by lack of response) that the upstream maintainer will make it easy for us to contribute this anyaway